FoodBookApp/UI/Views/CreateReview/CreateReview2View.swift

```diff
@@ -180,7 +192,7 @@ struct CreateReview2View: View {
         }
     }
     
-    var addPhotoButton: some View { // FIXME: should be component (?)
+    var addPhotoButton: some View { // FIXME: should be component (yes!)
```

Comment:
I'll  look into it since there's some back and forth communication between some variables. It can be an issue later